Resources/config/jackalope.xml

```diff
@@ -53,7 +53,7 @@
                  public="false"
         >
             <argument type="collection"/>
-            <argument type="service" id="doctrine.dbal.default_connection"/>
+            <argument type="service" id="doctrine_phpcr.jackalope_doctrine_dbal.default_connection"/>
```

Comment:
This is the change I've introduced, right? That's actually a shame, since I already had similar issues, but it seems like I've done it wrong on my own... Looks good to me.